gdx/src/com/badlogic/gdx/maps/tiled/BaseTmxMapLoader.java

```diff
@@ -400,6 +397,27 @@ protected void loadObject (TiledMap map, MapObjects objects, Element element, fl
 					object = new PolylineMapObject(polyline);
 				} else if ((child = element.getChildByName("ellipse")) != null) {
 					object = new EllipseMapObject(x, flipY ? y - height : y, width, height);
+				} else if ((child = element.getChildByName("text")) != null) {
+					TextMapObject textMapObject = new TextMapObject(x, flipY ? y - height : y, width, height, child.getText());
+					textMapObject.setRotation(child.getFloatAttribute("rotation", 0));
+					textMapObject.setFontFamily(child.getAttribute("fontfamily", ""));
+					textMapObject.setPixelSize(child.getInt("pixelSize", 16));
+					textMapObject.setHorizontalAlign(child.getAttribute("halign", "left"));
+					textMapObject.setVerticalAlign(child.getAttribute("valign", "top"));
+					textMapObject.setBold(child.getIntAttribute("bold", 0) == 1);
+					textMapObject.setItalic(child.getIntAttribute("italic", 0) == 1);
+					textMapObject.setUnderline(child.getIntAttribute("underline", 0) == 1);
+					textMapObject.setStrikeout(child.getIntAttribute("strikeout", 0) == 1);
+					textMapObject.setWrap(child.getIntAttribute("wrap", 0) == 1);
+					// When kerning is true, it won't be added as an attribute
+					textMapObject.setKerning(child.getIntAttribute("kerning", 1) == 1);
+					// Default color is #000000, not added as attribute
+					String textColor = child.getAttribute("color", "#000000");
+					// TODO:After PR #7076 is Merged. Update below section of code to use tiledColorToLibGDXColor().
+					String alpha = textColor.length() == 9 ? textColor.substring(1, 3) : "ff";
+					String color = textColor.length() == 9 ? textColor.substring(3) : textColor.substring(1);
+					textMapObject.setColor(Color.valueOf(color + alpha));
```

Comment:
There is a very similar code in `castProperty` that converts the type color to a LibGDX `Color` instance. Is `tiledColorToLibGDXColor` the extracted method for this?

Reply:
So in my TintColor PR. I made a convenience method that will be used in all the areas where we take in tiled colors turn them into the libgdx color format.  That's just a note to myself to replace it after that one get's merged. And yes its similar to the code in castProperty(). (Which is replaced in the TintColor PR). 

Though interestingly  the Color code generated in the map files for the custom "Color" type property is always in the full AARRGGBB format. Unlike some instances, like font color or tint color where the color is passed in as RRGGBB unless you modify the alpha values then it passes it in as AARRGGBB.  Confusing, I know. lol